frontend/public/components/_masthead.scss

```
@@ -1,3 +1,56 @@
+.co-app-launcher {
+  ul {
+    list-style: none;
+    margin-bottom: 0;
+    padding-left: 0;
+  }
+
+  .pf-c-app-launcher__menu-item-external-icon {
+    --pf-c-app-launcher__menu-item-external-icon--Color: var(--pf-global--icon--Color--light);
+
+    opacity: 1;
+  }
+}
+
+.co-cluster-menu {
+  max-width: 80vw;
+
+  @media (min-width: $screen-sm) {
+    max-width: 90vw;
+  }
+}
+
+.co-cluster-selector {
+  display: inline-grid !important;
+  grid-template-columns: 1fr auto;
+  max-width: 140px !important;
+
+  @media (min-width: 360px) {
+    max-width: 180px !important;
+  }
+
+  @media (min-width: $screen-xs) {
+    max-width: 300px !important;
+  }
+
+  @media (min-width: $screen-lg) {
```

```suggestion
  @media (min-width: $pf-global--breakpoint--xl) {
```
`$screen-lg` === `$pf-global--breakpoint--xl` => 1200

---

Leave as is.